### DISCUSSION TITLE
is dex dead?

### DISCUSSION THREAD

**Discussion by sourava01** [dexidp/dex#3244] [Q&A]
I want to use dex in one of my projects. However I see that docs are not updated. I cannot find features supported by dex in the documentation website. The docs are also very confusing with no or very less examples. There are also very fewer releases this year. 
So before I start using dex, I want to ask -  Is dex dead? Will be be seeing docs update in the future? Will we keep receiving updates in the foreseeable future?

**Comment by nabokihms** [comment]
The company I currently work for uses Dex in more than three hundred installations, and I know many more companies that do the same. The current problem is the number of active maintainers. Because of this, our primary focus is to keep Dex up-to-date without security holes than on the new features.

I can promise that we will still support the project, and as far as I know, neither of the current maintainers wants to quit.

**Reply by sourava01** [reply]
Cool! thanks for the clarification. would love to see this community grow!
any plans to improve the documentation? maybe atleast update it with all the features supported by dex. Would love to help here but I am pretty clueless myself on the feature list. went through hours of content just for implementing simple features.

**Reply by nabokihms** [reply]
There are no specific plans, but we understand that configuration and documentation are what users see first and what makes the first impression of the product, so we will work on these aspects.
